oecophylla/taxonomy/parser.py

```diff
@@ -73,6 +77,60 @@ def extract_level(table, code, delim=';', dic=None):
     return df
 
 
+def combine_kraken(kraken_reports):
+    """Combines Kraken-style reports for several samples into one table.
+
+    Parameters
+    ----------
+    kraken_reports : Iterable((str, str))
+        An iterable of tuples, where the second component is the filepath
+        pointing to a Kraken report file, the first component defines a
+        sample name for the Kraken report.
+
+    Returns
+    -------
+    tuple of (
+        Pandas.DataFrame
+            with rows for features, columns for samples
+        dict of set of str
+            level code : set of TaxIDs
```

Comment:
What are the keys if it is a dict of set of str?

Reply:
Thanks! "level code" is the key here. If this docstring is not clear enough, please let me know how I can improve it. @mortonjt